addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -408,6 +408,7 @@ def _message_post_after_hook(self, message, msg_vals):
             # on a document without customer means that it was created through the chatter using
             # suggested recipients. This heuristic allows to avoid ugly hacks in JS.
             new_partner = message.partner_ids.filtered(lambda partner: partner.email == self.email_from)
+            new_partner.type = 'private'
```

Comment:
According to the comment above, this partner is assumed to be a new partner.
But I'm really not sure about this.
What if we have a partner since a few years and he suddenly decides to apply? Now the existing partner just became private...